newspaper/static/utility.css

```diff
@@ -0,0 +1,5 @@
+:root{
+    --darkbrown:#272727;
+    --lightgray : #DFDFDF;
+
+}
```

Comment:
This is supposed to be defined in the same CSS if I am not wrong. Change colour names as well please, for example change `#272727` to something like `gmt-black` to know its a black shade and a not brown. And `lightgray` is intended as an alternate to white so something like `gmt-white-one` will be better.